lychee-lib/src/extract/html/html5gum.rs

```diff
@@ -507,4 +518,14 @@ mod tests {
         let uris = extract_html(input, false);
         assert_eq!(uris, expected);
     }
+
+    #[test]
+    fn test_skip_preconnect() {
+        let input = r#"
+            <link rel="preconnect" href="https://example.com">
```

Comment:
I suggest adding a testcase for the opposite order of attributes, `<link href=... rel=...>` -- not sure at the moment if it would pass

Reply:
This is fixed... finally. 🤣